src/card.tsx

```diff
@@ -486,6 +488,13 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
                                     buttonLink={buttonLink}
                                 />
                             </div>
+                            {icon ? (
+                                <Box className={styles.touchableCardOverlay} paddingX={16} paddingY={16}>
```

Comment:
This space in desktop is bigger (24), sorry the branch with this information is not merged in Figma specs document
![image](https://github.com/Telefonica/mistica-web/assets/6722153/6ef6e994-1c6e-45df-82a3-94f141e83089)